snapcraft/internal/parser.py

```diff
@@ -59,7 +63,11 @@ class MissingSnapcraftYAMLError(Exception):
 
 # TODO: make this a temporary directory that get's removed when finished
 BASE_DIR = os.getenv('TMPDIR', '/tmp')
-PARTS_FILE = "snap-parts.yaml"
+PARTS_FILE = 'snap-parts.yaml'
+DEFAULT_INDEX = 'http://wiki.ubuntu.com/snapcraft/parts?action=raw'
+
+__doc__ = __doc__.format(default_index=DEFAULT_INDEX,
+                         default_parts_file=PARTS_FILE)
```

Comment:
I agree that it's a neat trick, but there's a reason docopt uses the docstring for this-- so the documentation can be available to someone reading the code as _well_ as someone running `-h`. I appreciate the deduplication, but this makes it hard to read for the person in the code.